gcsfs/tests/test_core.py

```diff
@@ -702,12 +702,12 @@ def test_writable():
 def test_merge():
     with gcs_maker() as gcs:
         with gcs.open(a, "wb") as f:
-            f.write(b"a" * 100)
+            f.write(b"a" * 10)
```

Comment:
Why were these data sizes changed?

Reply:
test was consistently failing with rate exceeded error when executed under default user authentication. As far as I understand, actual value doesn't matter that much.

Reply:
Indeed it doesn't matter for this test, but it is worrying nevertheless. There should be the same number of requests either way, one would think... Never mind!